### ISSUE TITLE
目前成绩0.80510，使用修改过的LogisticModel

### ISSUE THREAD

**Issue by NiyunZhou** [NiyunZhou/The21-dayExpendables#37]
使用的模型位于Google_Cloud_dev中的LogisticModel。
training set + validation set
注意：dropout一定要使用is_training选项，这样dropout才会正常工作，降低过拟合。
lr 5e-4
batch_size 1024
dropout probability 0.5
其他参数均为默认
个人感觉提升每层的权重数目有助于提高准确率
GAP从0.77到0.805

名次一下上升了45名，目前83名。 

大家加油，冲进前50！！！
